apis/pom.xml

```diff
@@ -1,29 +1,67 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
+  <!-- Artifact props -->
   <groupId>io.stargate</groupId>
   <artifactId>sgv2-api-parent</artifactId>
-  <name>Stargate V2 API parent pom</name>
+  <name>Stargate - V2 API Parent pom</name>
   <version>2.0.10-SNAPSHOT</version>
-  <description>Parent POM for Stargate v2 APIs</description>
   <packaging>pom</packaging>
-  <url>http://github.com/stargate/stargate</url>
+  <!-- Description -->
+  <description>Stargate - An open source data API gateway</description>
+  <url>https://github.com/stargate/stargate</url>
+  <!-- Dev props -->
+  <developers>
+    <developer>
+      <id>stargate</id>
+      <name>The Stargate Authors</name>
+      <email>maintainers@stargate.io</email>
+      <organization>Stargate</organization>
+      <organizationUrl>https://stargate.io</organizationUrl>
+    </developer>
+  </developers>
+  <!-- License -->
+  <licenses>
+    <license>
+      <name>The Apache Software License, Version 2.0</name>
+      <url>https://www.apache.org/licenses/LICENSE-2.0.txt</url>
+      <distribution>repo</distribution>
+    </license>
+  </licenses>
+  <!-- Source code control management -->
+  <scm child.scm.connection.inherit.append.path="false" child.scm.developerConnection.inherit.append.path="false" child.scm.url.inherit.append.path="false">
+    <url>https://github.com/stargate/stargate</url>
+    <connection>scm:git:git://github.com/stargate/stargate.git</connection>
+    <developerConnection>scm:git:git://github.com/stargate/stargate.git</developerConnection>
+    <tag>HEAD</tag>
+  </scm>
+  <!-- Issue management -->
+  <issueManagement>
+    <system>GitHub</system>
+    <url>https://github.com/stargate/stargate/issues/</url>
+  </issueManagement>
```

Comment:
These main props are same in both root poms, maybe we should somehow have this shared somewhere.. Not sure how..

Reply:
I know the feeling, DRY and all. But for now maybe just leave duplicated?

Reply:
Yea will leave duplicated for now.